src/ServiceFabricPersistence/ServiceFabricPersistence.csproj

```diff
@@ -15,12 +15,11 @@
     <PackageReference Include="Microsoft.ServiceFabric.Data" Version="[3.0.456, 4.0)" />
     <PackageReference Include="Newtonsoft.Json" Version="[11.0.1, 12.0.0)" />
     <PackageReference Include="NServiceBus" Version="[7.0.0, 8.0.0)" />
-    <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="All" />
-    <PackageReference Include="Particular.Packaging" Version="0.1.0" PrivateAssets="All" />
+    <PackageReference Include="Particular.CodeRules" Version="0.2.1" PrivateAssets="All" />
+    <PackageReference Include="Particular.Packaging" Version="0.2.0" PrivateAssets="All" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="SourceLink.Create.GitHub" Version="2.6.0" PrivateAssets="All" />
     <DotNetCliToolReference Include="dotnet-sourcelink-git" Version="2.6.0" />
```

Comment:
When switching to Particular.Packaging 0.2.0, you should remove this entire `ItemGroup`.